recipes-kernel/linux/linux-hikey-aosp_4.4.bb

```diff
@@ -71,4 +71,6 @@ do_configure() {
     bbplain "Saving defconfig to:\n${B}/defconfig"
     oe_runmake -C ${B} savedefconfig
     cp -a ${B}/defconfig ${DEPLOY_DIR_IMAGE}
+    bbplain "Saving .config to:\n${B}/config"
+    cp -a ${B}/.config ${DEPLOY_DIR_IMAGE}/config
```

Comment:
first.. it is wrong to 'deploy' files from do_configure(), but it's not your fault, since you copy/pasted a bad code that was already doing it ;-)

why do you need the full config, since we already have the minimal defconfig?